tests/e2e/support/environment/actor/shared.ts

```diff
@@ -20,6 +20,7 @@ export interface ActorOptions extends ActorsOptions {
 export const buildBrowserContextOptions = (options: ActorOptions): BrowserContextOptions => {
   const contextOptions: BrowserContextOptions = {
     acceptDownloads: options.context.acceptDownloads,
+    permissions: ['clipboard-read', 'clipboard-write'],
```

Comment:
Can we only open read permission as we don't need write permissions as of now.
```suggestion
    permissions: ['clipboard-read'],
```